### PR TITLE
[admin] Disconnect panel

### PR DESCRIPTION
**Pull request by adamsong**
# Github documenting your Pull Request
Adds a 'Disconnect Panel' which lists all players who have at some point disconnected, giving information about their connection history. This can be used to find players who have been disconnected for an extended period of time, and deal with them.

image

# Changelog

:cl:  
rscadd: Added a panel to show connection information about players.
/:cl:

### CODE REVIEW

**Comment by adamsong** [comment]
It says they are not intended the same, but they are indented the same, anyone can help?

File: code/modules/admin/verbs/disconnectpanel.dm [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+/datum/connection_entry
+	var/disconnected
+	var/connected
+	var/disconnect_type
+	var/living = FALSE
+	var/job
+
+/client/proc/disconnect_panel()
+	set name = "Disconnect Panel"
+	set desc = "Panel showing information about the currently disconneted players"
```
Comment (Hopekz) [inline code/modules/admin/verbs/disconnectpanel.dm]:
```suggestion
	set desc = "Panel showing information about the currently disconnected players"
```

**Review by Hopekz — CHANGES_REQUESTED** [review]

**Comment by N3D6** [comment]
Really based PR, makes it a lot easier to see who is disconnected and for how long.

**Review by JamieD1 — APPROVED** [review]
Works well.

**Pull request merged**